resources/logging/values.yaml

```diff
@@ -1,82 +1,146 @@
-test:
-  image:
-    name: test-logging
-
-logspout:
-  resources:
-    requests:
-      cpu: 125m
-      memory: 96Mi
-    limits:
-      memory: 400Mi
-  image:
-    repository: gliderlabs/logspout
-    tag: "v3.2.5"
-    pullPolicy: IfNotPresent
-  tolerations:
-  - effect: NoSchedule
-    operator: Exists
+rbac:
+  create: true
+  pspEnabled: true
 
-oklog:
-  image:
-    repository: oklog/oklog
-    tag: "v0.3.2"
-    pullPolicy: IfNotPresent
-  # The number of log collection service replicas. Should be a minimum of 3.
+serviceAccount:
+  create: true
+  name: 
+
+loki: 
   replicas: 1
+  deploymentStrategy: RollingUpdate
+
+  image:
+    repository: grafana/loki
+    tag: master
```

Comment:
please switch to a hash value rather than using "master"

Reply:
docker image tags changed to testet version